docs/book/user-guide/cloud-guide/aws-guide.md

```diff
@@ -0,0 +1,201 @@
+
+To follow this guide, you need:
+
+- An active AWS account with necessary permissions for AWS S3, SageMaker, and ECR.
+- ZenML [installed](../../getting-started/installation.md)
+- AWS CLI installed and configured with your AWS credentials. You can follow the instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html).
+
+Once ready, navigate to the AWS console:
+
+1. Choose an AWS region
+In the AWS console, choose the region where you want to deploy your ZenML stack resources. Make note of the region name (e.g. `us-east-1`, `eu-west-2`, etc.) as you will need it in subsequent steps.
+
+1. Create an IAM user
+Create a new IAM user that ZenML will use to access AWS resources:
+
+- Go to the IAM dashboard in the AWS console
+- Click on "Users" in the left sidebar
+- Click "Add user"
+- Enter a user name like zenml-user
+- Select "Programmatic access" under "Access type"
+- Click "Next: Permissions"
+- Click "Attach existing policies directly"
+- Search for and select the following policies:
+  - AmazonS3FullAccess
+  - AmazonECS_FullAccess
+  - AmazonECRFullAccess
+- Click "Next: Tags"
+- Optionally add tags and click "Next: Review"
+- Click "Create user"
+- Click "Download .csv" to save the access key ID and secret access key. You will need these in the next step.
+
+{% hint style="warning" %} Keep the access keys secure and do not share them publicly as they allow full access to your AWS resources! {% endhint %}
+
+3. If you have not already, install the AWS and S3 ZenML integrations:
+
+```shell
+zenml integration install aws s3 -y
+```
+
+## 2) Create a Service Connector within ZenML
+
+Create an AWS Service Connector within ZenML the service connector will allow ZenML and other ZenML components to authenticate themselves with AWS.
+
+{% tabs %}
+{% tab title="CLI" %}
+
+```shell
+zenml service-connector register aws_connector \
+  --type aws \
+  --auth-method secret-key \
+  --aws_access_key_id=<YOUR_ACCESS_KEY_ID> \  
+  --aws_secret_access_key=<YOUR_SECRET_ACCESS_KEY> \
+  --region=<YOUR_REGION>
+```
+
+Replace <YOUR_ACCESS_KEY_ID>, <YOUR_SECRET_ACCESS_KEY> and <YOUR_REGION> with the respective values.
+{% endtab %}
+{% tab title="Dashboard" %}
+<figure><img src="../../.gitbook/assets/aws-service-connector-type.png" alt=""><figcaption><p>Create the service connector easily in the dashboard</p></figcaption></figure>
+
+You can use the "Connectors" section of the ZenML dashboard to create a AWS service connector in a no-code manner.
+{% endtab %}
+{% endtabs %}
+
+## 3) Create Stack Components
+
+### Artifact Store (S3)
+
+An [artifact store](../production-guide/remote-storage.md) is used for storing and versioning data flowing through your pipelines.
+
+1. Before you run anything within the ZenML CLI, head on over to AWS and create a AWS bucket, in case you don't already have one that you can use. 
+
+```shell
+aws s3api create-bucket --bucket your-bucket-name --region your-region-name
+```
```

Comment:
one or the other. i.e. just use the CLI command, or just pick one you already have.